salt/orchestrate/aws/cloud_profiles/backup_host.conf

```diff
@@ -2,6 +2,7 @@ backup_host:
   provider: mitx
   size: t2.medium
   image: ami-49e5cb5e
+  availability_zone: us-east-1b
```

Comment:
I don't think we need to specify the AZ because that would be implicit from the subnet that it gets placed into.